command/agent/auth/azure/azure.go

```diff
@@ -176,3 +175,33 @@ func getMetadataInfo(ctx context.Context, endpoint, resource string) ([]byte, er
 
 	return body, nil
 }
+
+func verifyIdentity(ctx context.Context) (retPath string, header http.Header, retData map[string]interface{}, retErr error) {
```

Comment:
This function doesn't seem to be called anywhere, where should this be called? Does the return values need to be named here?